content/20.treatments.md

```diff
@@ -193,6 +193,8 @@ HCQ and CQ act through the same mechanism of action, but HCQ is regarded to be s
 HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
 _In vitro_ studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
 In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+A case study of three COVID-19 patients reported presence of antiphospholipid antibodies in all three patients  [@doi:10.1056/NEJMc2007575]. Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome. HCQ has been reported to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023;doi:10.1016/j.autrev.2014.01.053;doi:10.1182/asheducation-2016.1.714]. 
```

Comment:
```suggestion
A case study of three COVID-19 patients reported presence of antiphospholipid antibodies in all three patients  [@doi:10.1056/NEJMc2007575]. 
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome. 
HCQ has been reported to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
```
Reformating for one sentence per line (weird as a writer, great for tracking changes in GitHub!) and the multi-paper citations.